lib/cdo/properties.rb

```diff
@@ -40,19 +40,16 @@ def self.get_user_metrics()
 end
 
 def fetch_metrics()
-  # TODO(asherkach): Reenable Properties.get(:metrics). It is being disabled
-  # as we are seeing 0 lines of code being returned.
-  # Properties.get(:metrics) || {
-  {
-    'created_at'=>"2015-11-04T21:11:40+00:00",
-    'created_on'=>"2015-11-04",
+  Properties.get(:metrics) || {
```

Comment:
Please add a comment explaining the the Properties.get returns nil when S3 is unaccessible and that we fall back on stale historical values for that case.